uitools/import/Esri/ArcGISRuntime/Toolkit/Callout.qml

```diff
@@ -473,11 +488,17 @@ Item {
                     rows: 2
                     columnSpacing: 7
 
-                    Rectangle {
+                    Pane {
```

Comment:
❌  Honestly, I won't touch Callout at all. There's too much else wrong with it.

Plus if we are setting the Pane, then hard-overwriting the color, then there was no point setting Pane at all, we should have left this as a Rectangle.